ts/src/core/query/query.ts

```diff
@@ -125,7 +128,7 @@ function assertValidCursor(cursor: string, opts: validCursorOptions): any {
   if (parts.length !== keys.length * 2) {
     throw new Error(`invalid cursor ${cursor} passed`);
   }
-  const values: any = [];
+  const values: [key: string, value: string | number | null][] = [];
```

Comment:
wowza. what's this type?

is this a fancy TS tuple? i had forgotten this could be done... I guess I don't write enough TS anymore

Reply:
Yeah, it's just a tuple, but you can name each piece in TS now.